src/main/java/com/gooddata/account/AccountService.java

```diff
@@ -49,4 +50,57 @@ public void logout() {
             throw new GoodDataException("Unable to logout", e);
         }
     }
+
+    /**
+     * Creates new account in given organization (domain)
+     * @param account to create
+     * @param organizationName (domain) in which account should be created
+     * @return new account
+     */
+    public Account createAccount(Account account, String organizationName) {
+        notNull(account, "account");
+        notNull(organizationName, "organizationName");
+
+        try {
+            final UriResponse uriResponse = restTemplate.postForObject(Account.CREATE_URI, account, UriResponse.class, organizationName);
+            return getAccount(Account.getId(uriResponse.getUri()));
+        } catch (GoodDataException | RestClientException e) {
+            throw new GoodDataException("Unable to create account", e);
+        }
+    }
+
+    /**
+     * Delete given account
+     * @param account to remove
+     */
+    public void removeAccount(Account account) {
+        notNull(account, "account");
+
+        try {
+            restTemplate.delete(account.getUri());
+        } catch (GoodDataRestException e) {
+            if (HttpStatus.NOT_FOUND.value() == e.getStatusCode()) {
+                throw new AccountNotFoundException(account.getUri(), e);
+            } else {
+                throw e;
+            }
+        } catch (GoodDataException e) {
+            throw new GoodDataException("Unable to remove account", e);
+        }
+    }
+
+    private Account getAccount(String id) {
```

Comment:
because I was lazy to write tests and documentation for  it :D ... but since you caught me red handed I made it public :)